src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Logging/ISuppressableLog.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.DotNet.ApiCompatibility.Logging
     public interface ISuppressableLog : ILog
     {
         /// <summary>
-        /// Reports whether the logger emitted a compatibility suppression.
+        /// Reports whether the logger emitted an error suppression.
```

Comment:
Ah, good to know. Based on your PR description:

> _APICompat logs a success or error message in case breaking changes are found. It currently also logs an error if a warning is logged. Change that so that a warning doesn't impact the compatibility result._

I was going to ask if a warning was considered a compatibility issue, but it seems that's not the case. Correct?

Reply:
Correct - that's the intent of this change :)